### PR TITLE
refactored with eslint

### PR DESCRIPTION
**Pull request by meltedspork**
To be merged into feature/swagger-docker - cleaned up with eslint code quality

### CODE REVIEW

File: src/api/category.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
-module.exports.register = (server) => {
+function register (server) {
```
Comment (kratsg) [inline src/api/category.js]: any reason why it moved away from double-arrow notation here?

File: src/services/logger.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import bunyan from 'bunyan'
+import Bunyan from 'bunyan'
```
Comment (kratsg) [inline src/services/logger.js:1]: capitalized for some reason?

**Review by kratsg — CHANGES_REQUESTED** [review]

File: src/api/category.js (hunk 1 above)
Comment (meltedspork) [reply src/api/category.js]: Oh none, actually! I will switch it to arrow function 👍

File: src/services/logger.js (hunk 2 above)
Comment (meltedspork) [reply src/services/logger.js:1]: Any time when we do new constructor, we should be doing capitalize 
`[eslint] A constructor name should not start with a lowercase letter. (new-cap)`
 See: https://eslint.org/docs/rules/new-cap for more details

File: src/config/database.js [hunk 3]
```diff
@@ -1,3 +1,6 @@
+// NOTE: for some reason the SignsFive API does
+// not like 'export default' being used here.
+// So, would like to dig in this sometime other time.
 module.exports = {
```
Comment (meltedspork) [inline src/config/database.js:4]: I am not sure what is going on with this. I kept getting but that might be due to being used by sequelize before starting app. is that possible?

**Review by kratsg — APPROVED** [review]

File: src/config/database.js (hunk 3 above)
Comment (meltedspork) [reply src/config/database.js:4]: @kratsg  ^^

Comment (kratsg) [reply src/config/database.js:4]:
```
db/models/index.js
8:var config    = require(__dirname + '/../../config/database.js')[env];
14:  var sequelize = new Sequelize(config.database, config.username, config.password, config);
```

Comment (kratsg) [reply src/config/database.js:4]: So I guess figuring out how to grab the config here should be improved.

Comment (meltedspork) [reply src/config/database.js:4]: Yeah, not high priority for one small file. 👍

**Pull request merged**